buildkite/bazelci.py

```diff
@@ -336,6 +336,10 @@
 EDql
 """.strip()
 
+BAZELISK_VERSION_ENV_VAR = "USE_BAZEL_VERSION"
```

Comment:
If it's only used once in the whole script, I think it's fine to just use the literal string below.